### ISSUE TITLE
Cannot edit builds and notes at the same time

### ISSUE THREAD

**Issue by bowlofeggs** [fedora-infra/bodhi#2041]
@AdamWill reported on Freenode:

```
<adamw> bowlofeggs: i seem to be having rather a lot of trouble getting edits to update texts to work at present?
<adamw> any know issues?
<adamw> for e.g. i just edited https://bodhi.fedoraproject.org/updates/FEDORA-2017-2b39d3fe2d , changed it from fedfind-3.8.2-1.fc27 to fedfind-3.8.3-1.fc27 and edited the text; the change of package build took effect, but the text doesn't seem to have changed
<bowlofeggs> adamw: hmm that is odd - that doesn't sound like a bug report i'm familiar with
<bowlofeggs> adamw: is it repeatable you think?
<adamw> bowlofeggs: it happened like five times
<adamw> i have fedfind updates for five releases, every single one behaved the same...
```

**Comment by AdamWill** [comment]
Couple of notes: 

* I did try just editing the text for some of the relevant updates, and that didn't seem to work either.
* I've done this many times in the past without trouble.
* After the edit operation, the page didn't automatically refresh and show me the edited update as it usually would, but reloading it manually showed the new build in the update.

**Comment by AdamWill** [comment]
One more thing: the titles of the edited updates - as shown in Bodhi's summary views, like "fedfind-3.8.2-1.fc27 python-wikitcms-2.2.1-1.fc27 relval-2.2.0-1.fc27" - still refer to the *old* build (as you can see there, it says 3.8.2-1.fc27, not 3.8.3-1.fc27). But the 'builds' tab on each update shows the new build. Also, the usual 'events' that happen when you change builds in an update - those bodhi-sourced comments, like "adamwill edited this update. New build(s): [foo] Removed build(s): [foo] Karma has been reset. This update has been submitted for testing by adamwill. This update has been pushed to testing." do not seem to have happened. So the updates really seem to be stuck in a weird state :( I've no idea what to do with them now.

**Comment by bowlofeggs** [comment]
I tried to reproduce what you are seeing with one of my updates, but I was able to edit the text without an issue: https://bodhi.fedoraproject.org/updates/FEDORA-2017-bb746cfd77

You mentioned that the old builds are still shown in the title - there's an open bug about that: https://github.com/fedora-infra/bodhi/issues/1946

I don't have time to investigate more deeply atm, but I noticed that the edit form redirected me to this really nasty URL: https://bodhi.fedoraproject.org/updates/ejabberd-17.09-1.fc27%20erlang-cache_tab-1.0.11-1.fc27%20erlang-esip-1.0.16-1.fc27%20erlang-fast_tls-1.0.16-1.fc27%20erlang-fast_xml-1.1.24-1.fc27%20erlang-fast_yaml-1.0.11-1.fc27%20erlang-iconv-1.0.6-1.fc27%20erlang-p1_oauth2-0.6.2-1.fc27%20erlang-p1_utils-1.0.10-1.fc27%20erlang-stringprep-1.0.10-1.fc27%20erlang-stun-1.0.15-1.fc27%20erlang-xmpp-1.1.15-1.fc27

I wonder if there is something funny going on with that title field. To be honest, I kinda want to get rid of the title as an identifying field for updates because we already have the alias (the FEDORA-2017-abcde style identifier) and it's silly to have both.

**Comment by AdamWill** [comment]
So the affected updates are still screwed up (except the F26 one which somehow managed to get pushed successfully), and it's kind of a big problem for me. Can someone please just straighten them out by hand, or something? They are:

* [This one for F27](https://bodhi.fedoraproject.org/updates/FEDORA-2017-2b39d3fe2d)
* [This one for EPEL 7](https://bodhi.fedoraproject.org/updates/FEDORA-EPEL-2017-c0be6372ac)
* [This one for EPEL 6](https://bodhi.fedoraproject.org/updates/FEDORA-EPEL-2017-a96a15e55c)

I want all of them pushed stable, with the builds listed on the 'Builds' tab (i.e. fedfind 3.8.3, not 3.8.2). I submitted them all for stable, and they all got ejected from the push due to the fedfind package not having the right tag, I believe because it was never actually pushed to updates-testing after I submitted it.

**Comment by bowlofeggs** [comment]
Hmm yeah that is super weird. Looking at the first one, for some reason fedfind is tagged wtih 
f27-updates-candidate even though Bodhi thinks it is in the testing repo.

I think we will need a release engineer to get involved because I don't have permissions to mess with Koji tags. Let's discuss in #fedora-releng.

**Comment by bowlofeggs** [comment]
I don't think this is a Bodhi bug, or at least not the f27 one. I used koji list-tag-history to see that Bodhi did not put the build into this state:

```
$ koji list-tag-history --build fedfind-3.8.3-1.fc27
Tue Nov 21 17:15:42 2017: fedfind-3.8.3-1.fc27 tagged into f27-updates-candidate by adamwill [still active]
Thu Dec  7 17:50:42 2017: fedfind-3.8.3-1.fc27 tagged into f27-updates-pending by bodhi
Thu Dec  7 18:21:04 2017: fedfind-3.8.3-1.fc27 tagged into f27-override by bodhi
Fri Dec  8 19:00:04 2017: fedfind-3.8.3-1.fc27 untagged from f27-override by bodhi
Sun Dec 10 12:53:33 2017: fedfind-3.8.3-1.fc27 tagged into f27-signing-pending by puiterwijk
Sun Dec 10 12:53:38 2017: fedfind-3.8.3-1.fc27 tagged into f27-updates-testing-pending by autopen
Sun Dec 10 12:53:38 2017: fedfind-3.8.3-1.fc27 untagged from f27-signing-pending by autopen
Sun Dec 10 12:54:12 2017: fedfind-3.8.3-1.fc27 untagged from f27-updates-testing-pending by puiterwijk
Sun Dec 10 13:06:59 2017: fedfind-3.8.3-1.fc27 untagged from f27-updates-pending by bodhi
```

**Comment by bowlofeggs** [comment]
After talking with @puiterwijk I now do think it's a Bodhi bug - it lokos like the build never got tagged into testing even though Bodhi thinks the update is testing. I'm not sure how that happened yet.
